src/prepack-cli.js

```diff
@@ -196,7 +196,8 @@ function run(
         foundFatal = foundFatal || error.severity === "FatalError";
         console.log(
           `${loc.source || ""}(${loc.start.line}:${loc.start.column +
-            1}) ${error.severity} ${error.errorCode}: ${error.message}`
+            1}) ${error.severity} ${error.errorCode}: ${error.message}` +
+            ` (github.com/facebook/prepack/wiki/${error.errorCode})`
```

Comment:
Will this URL automatically turn into a hyperlink while running in prepack.io/repl.html? Do we need to add "https://" prefix per discussion below?
https://stackoverflow.com/questions/29681775/how-to-create-hyperlinks-linked-to-javascript-functions-in-chromes-console-log

Reply:
Ah I see, yes I'll add the https prefix.